.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: ooTjAQGAQVnXBEF6N5A0m7sNJeD30ieS0
```

Comment:
I thought you were adding this in the `.yml` file so no need to expose it.
Or it must be there?

I see you also `gitignored` it but it's still in version control.

Reply:
its in `gitignore`, i will have to force remove it

Reply:
If you delete it locally and commit the deletion, it'll delete it in version control. @oyinda-subair

Reply:
yeah thats what i meant 😄